test/extensions/INTEL/SPV_INTEL_fpga_buffer_location/FPGABufferLocation.ll

```diff
@@ -1,13 +1,13 @@
-; RUN: llvm-as %s -o %t.bc
-; RUN: llvm-spirv %t.bc --spirv-ext=+SPV_INTEL_fpga_buffer_location -o %t.spv
+; RUN: llvm-as -opaque-pointers=0 %s -o %t.bc
```

Comment:
I think we'd better stay with the opaque pointers enabled approach whenever possible

Reply:
Tried to replace"-opaque-pointers=0" with "-emit-opaque-pointers", it seems it does not work anymore. Do I need to change all pointers with "ptr" to get it work?

Reply:
done